packages/astro/src/content/vite-plugin-content-assets.ts

```diff
@@ -129,22 +129,9 @@ export function astroConfigBuildPlugin(
 	options: StaticBuildOptions,
 	internals: BuildInternals
 ): AstroBuildPlugin {
-	let ssrPluginContext: Rollup.PluginContext | undefined = undefined;
 	return {
 		targets: ['server'],
 		hooks: {
-			'build:before': ({ target }) => {
-				return {
-					vitePlugin: {
-						name: 'astro:content-build-plugin',
-						generateBundle() {
-							if (target === 'server') {
-								ssrPluginContext = this;
-							}
-						},
-					},
-				};
-			},
```

Comment:
All this `'build:before'` hook as doing was assigning to the `ssrPluginContext` variable... which was then never used. So I believe removing it altogether is safe, right?